src/components/duet-date-picker/date-picker-day.tsx

```diff
@@ -35,45 +35,20 @@ export const DatePickerDay: FunctionalComponent<DatePickerDayProps> = ({
     onDaySelect(e, day)
   }
 
-  if (disabled) {
-    return (
-      <span
-        class={{
-          "duet-date__day": true,
-          "is-outside": isOutsideRange,
-          "is-disabled": disabled,
-          "is-today": isToday,
-          "is-month": isMonth,
-        }}
-        role="button"
-        tabIndex={isFocused ? 0 : -1}
-        onKeyDown={onKeyboardNavigation}
-        aria-pressed="false"
-        aria-disabled="true"
-        ref={el => {
-          if (isFocused && el && focusedDayRef) {
-            focusedDayRef(el)
-          }
-        }}
-      >
-        <span aria-hidden="true">{day.getDate()}</span>
-        <span class="duet-date__vhidden">{dateFormatter.format(day)}</span>
-      </span>
-    )
-  }
-
   return (
     <button
       class={{
         "duet-date__day": true,
         "is-outside": isOutsideRange,
         "is-today": isToday,
         "is-month": isMonth,
+        "is-disabled": disabled,
       }}
       tabIndex={isFocused ? 0 : -1}
       onClick={handleClick}
       onKeyDown={onKeyboardNavigation}
-      disabled={isOutsideRange}
+      aria-disabled={disabled ? "true" : undefined}
+      disabled={isOutsideRange || !isMonth}
```

Comment:
> Working with the focused day and handling current year/month stuff should not be the developer's concern, this logic will always be required, so the picker should handle it :)

my use case is to get rid of the `isMonth` check, actually :-)

we're using the date picker in an absence management tool and days are coloured differently when it is a vacation day or a sick day for example. Currently we have no visual difference between the last day of the month and the first day of the next month. And I prefer to keep it like that :-) The user sees the days, so why should the tool prohibit selecting it?

for a lib I think it is ok to say "you have to handle everything yourself, if you are overriding the default behaviour." The date picker has a nice simple default of just picking dates, and can be fully enhanced by power users.

Reply:
Oh do you mean as it behaves here https://www.duetds.com/components/date-picker/

You can select any dates that show from the previous/next months. If so I simply forgot to to port that behaviour here and I'll make that change